src/StakingClient.ts

```diff
@@ -0,0 +1,174 @@
+// @ts-ignore
+import { liquidationStaking as backend } from '@xbacked-dao/xbacked-contracts';
+import { Account } from './Account';
+import { AbiInterface, AccountInterface, StakeGlobalView, StakeLocalView } from './interfaces';
+
+export class StakingClient extends Account {
```

Comment:
@AwolDes 
should the name for this be `StakingClient` or `StakingAccount`, if you want this to be `StakingClient`, then you could change the name of the Account class to `Client`

Reply:
I think `Account` is still a good name for this class, and the clients `extend` the account class. I think `Account` is more accurate than `Client`